solar-system.rb

```diff
@@ -0,0 +1,89 @@
+
+class SolarSystem
+  attr_reader :ss_user_choice
+
+  def initialize(ss)
+    @ss = ss
+  end
+
+
+  def ss_user_choice
+    puts "Here are your planets:"
+    @ss.each do |ss_planet|
+      puts ss_planet.name
+    end
+    puts "What planet do you want to learn more about? Please properly capitalize:"
+    # currently only accepts capitalized planets
+    planet_choice = gets.chomp
+    @ss.each do |ss_planet|
+      if planet_choice == ss_planet.name
+        puts ss_planet.print_planet
+      # else statement is always printing. not sure why
+      # else
+      #   puts "Sorry, you inputted an invalid planet"
+      #   exit
+      end
+    end
+  end
+end
+
+class Planet
+  attr_accessor :name, :moons, :year_length, :distance_from_the_sun, :named_after, :print_planet
```

Comment:
Again, you don't need to add `print_planet` to an `attr_accessor`. `print_planet` is the name of a method, which can already be called with `.print_planet`. `attr_accessor`s are for instance variables